classes/controller/Controller.php

```diff
@@ -180,9 +180,6 @@ public function init()
             define('_PS_BASE_URL_SSL_', Tools::getShopDomainSsl(true));
         }
 
-        if (null === $this->getContainer()) {
-            $this->container = $this->buildContainer();
-        }
         $localeRepo = $this->get(self::SERVICE_LOCALE_REPOSITORY);
```

Comment:
you call the container before its build, isn't it ?

Reply:
yes, I call the container builder so that its built. That's what was done, I just moved this process from the init() method to the constructor.